runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -2645,20 +2637,12 @@ J9::ARM64::TreeEvaluator::ArrayStoreCHKEvaluator(TR::Node *node, TR::CodeGenerat
       if (translatedNode->getOpCode().isRightShift()) // optional
          translatedNode = translatedNode->getFirstChild();
 
-      bool usingLowMemHeap = false;
-      if (TR::Compiler->vm.heapBaseAddress() == 0 || sourceChild->isNull())
-         usingLowMemHeap = true;
+      usingCompressedPointers = true;
```

Comment:
Similarly here.

Reply:
Similar question as in https://github.com/eclipse/openj9/pull/11688/files#r560678679 about this one.